clang/test/CodeGen/aarch64-elf-pauthabi.c

```diff
@@ -0,0 +1,59 @@
+// RUN: %clang --target=aarch64-linux -S -emit-llvm -o - \
```

Comment:
We prefer `%clang_cc1` for codegen tests. `%clang` is not used in clang/test/Driver

Reply:
Fixed, thanks, see fdb26a14024f5f8297480a74982a37ee988cd30f